python/BioSimSpace/Align/_align.py

```diff
@@ -38,6 +38,7 @@
 import subprocess as _subprocess
 import sys as _sys
 import tempfile as _tempfile
+import itertools
```

Comment:
As a convention, all external modules are listed alphabetically and imported privately so that they aren't exposed directly to the user, even if they import `_align.py` directly, e.g:

```python
import itertools as _itertools
```

This makes it obvious what  functionality comes from BioSImSpace when you import anything, i.e. only the top-level API of BioSimSpace is exposed, no private functions, external modules, variables, etc.

Reply:
see f93dfa096da5bb4be09259edeab90a2f1f77c767